deploy/3.rateOracles.ts

```diff
@@ -120,7 +120,6 @@ const deployAndConfigureWithGenerator = async ({
       from: deployer,
       args: args,
       log: doLogging,
-      gasLimit: 20000000,
```

Comment:
@0xZenus Do you understand why we had this? I found it made everything unrealistically expensive so that it was unaffordable with the deployer's current ETH balance. I've not had problems with an implicit gas limit but I assume we hit some and that's why we added this line?

Reply:
Hmm, I don't remember. However, it's fine letting it like this and increasing with 5-10% if we encounter any issues.